aplib.net-demo/Assets/Scripts/Player/Movement.cs

```diff
@@ -133,11 +134,10 @@ private void LimitVelocity(bool isOnWalkableSlope)
         }
         else
         {
-            Vector3 rigidbodyHorizontalVelocity = new(_rigidbody.velocity.x, 0, _rigidbody.velocity.z);
-            if (rigidbodyHorizontalVelocity.magnitude > _maxSpeed)
+            if (_rigidbody.velocity.magnitude > _maxSpeed && _isGrounded)
             {
-                Vector3 limitedVelocity = rigidbodyHorizontalVelocity.normalized * _maxSpeed;
-                _rigidbody.velocity = new Vector3(limitedVelocity.x, _rigidbody.velocity.y, limitedVelocity.z);
+
+                _rigidbody.velocity = _rigidbody.velocity * 0.95f;
```

Comment:
You should also define this 0.95f somewhere.
Also, please update the method summary to align with your changes.